### PR TITLE
feat: done ques page

### PR DESCRIPTION
**Pull request by Developers-codz**
Done with question pages
Thank you for reviewing my PR :)

### CODE REVIEW

**Review by ArjunJunna — APPROVED** [review]
Was looking for the link till now... Everything else looks good...

**Comment by Developers-codz** [comment]
> Was looking for the link till now... Everything else looks good...

File: styles/question.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
+.question-wrapper {
+  min-height: 55vh;
+  text-align: center;
+  margin: 3rem;
+}
```
Comment (foolhardy21) [inline styles/question.css:5]: just a suggestion to improve code readability. add a line space after a selector.

**Review by foolhardy21 — APPROVED** [review]
rest looks good to merge.

**Pull request merged**